xbmc/rendering/dx/DirectXHelper.cpp

```diff
@@ -9,6 +9,7 @@
 #include <d3d11_4.h>
 #include <dxgicommon.h>
 #include <dxgiformat.h>
+#include <libavutil/pixfmt.h>
```

Comment:
As FFmpeg source is C (not C++) headers should be added like:

```c++
extern "C"
{
#include <libavutil/pixfmt.h>
}
```

Anyway I'm thinking is better to move this to `RendererSoftware` that already has these headers

Reply:
noted, most places use that but quite a few don't